paywall/src/paywall-builder/config.js

```diff
@@ -26,6 +30,26 @@ export function setupReadyListener(window, iframe, origin) {
       // <script type="text/javascript">window.unlockConfig = {...}</script>
       // immediately before paywall.min.js is loaded
       sendConfig(window.unlockConfig, iframe, origin)
+      if (window.ethereum) {
+        window.ethereum.enable().then(() => {
```

Comment:
I think not all web3 provider actually implement `.enable`. We probably want to check if it exists before.

Reply:
good catch!